### PR TITLE
GUA-209: Save form data to session, make it a VC and persist to ESS

### PR DESCRIPTION
**Pull request by huwd**
WIP - see commit messages for detail

### CODE REVIEW

File: src/components/vocabularies/IdentityCheckCredential.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export type IdentityCheckCredentialType =
   | "IdentityCheckCredential"
   | "VerifiableCredential";
 
-export interface Credentials {
+export interface Credential {
```
Comment (huwd) [inline src/components/vocabularies/IdentityCheckCredential.ts]: Ah interesting, yeah that makes more sense.
FYI we'd this means we're diverging from the JSON Schema at https://vocab.london.cloudapps.digital/

Probably OK as if we tried to regenerate these we'd lose the refactor of common elements out anyway.

File: package.json [hunk 2]
```diff
@@ -51,6 +51,7 @@
     "@types/express": "^4.17.13",
     "@types/http-errors": "^1.8.2",
     "@types/i18next-fs-backend": "^1.1.2",
+    "@types/jsonwebtoken": "^8.5.8",
```
Comment (huwd) [inline package.json:54]: Ah great shout, I forgot that. We can squash these down with the previous when we're done perhaps?

File: src/config.ts [hunk 3]
```diff
@@ -29,6 +29,11 @@ export function getClientId(environment?: string) {
   return `${hostname}/info/id`
 }
 
+export function getJwtSigningKey() {
+  // This is a prototype, so we're not worried about the security of our key
```
Comment (huwd) [inline src/config.ts:33]: 👍🏻

File: src/components/vocabularies/IdentityCheckCredential.ts (hunk 1 above)
Comment (alex9smith) [reply src/components/vocabularies/IdentityCheckCredential.ts]: We talked about this and decided it was ok - the published vocab is in need of a bit of love anyway so diverging from it is fine.

**Review by alex9smith — APPROVED** [review]

**Pull request merged**